ccf_transaction_processor/README.md

```diff
@@ -260,3 +262,40 @@ make -C ${PDO_SOURCE_ROOT}/build force-conf keys
 cd ${PDO_SOURCE_ROOT}/build
 make test
 ```
+## CCF based PDO TP deployment on SGX enabled HW
+
+As per CCF documentation (https://microsoft.github.io/CCF/ccf-0.11.7/quickstart/index.html), CCF with full security guarantees requires SGX hardware with FLC. Below we provide instructions to deployment for CCF on SGX HW via docker containers running on Azure Confidential Compute (ACC) VMs. For instructions on how to deploy an ACC VM suitable for CCF installation, please see https://microsoft.github.io/CCF/ccf-0.11.7/quickstart/index.html. Essentially, the only requirement from the host platform is that the DCAP aware SGX driver (See https://github.com/intel/SGXDataCenterAttestationPrimitives/tree/master/driver/linux) is installed and loaded. If installed, the device nodes can be found at /dev/sgx (/dev/sgx/enclave and /dev/sgx/provision). The rest of the dependencies required to deploy PDO TP will be automatically taken care of within the docker container.
+
+Note: The instructions below have been tested only on ACC VMs and not on other SGX enabled platforms.
+
+Set the env variable HOST_IP to the ip-address of the host machine. (The docker container will use HOST networking mode). Run the following commands to build and deploy a single node docker container running PDO TP that is available for business transactions at http://HOST_IP:6600
```

Comment:
Why do you _force_ `host` network?  That prevents running more than one tp per host and hence also prevents (easy) integration-tests of multi-ccf installations?